### PR TITLE
PW-1510 Add instalments for Terminal API

### PR DESCRIPTION
**Pull request by cyattilakiss**
**Description**
Add configuration for POS instalments
Add POS instalments
Refactor instalments on the frontend (credit cards, pos cloud)
If no terminal is available show message

### CODE REVIEW

File: etc/adminhtml/system/adyen_pos_cloud.xml [hunk 1]
```diff
@@ -78,6 +78,28 @@
             <config_path>payment/adyen_pos_cloud/recurring_type</config_path>
             <comment>If you want to store the credentials for ONECLICK or RECURRING or combination of these two you need to contact support@adyen.com and ask if they can enable to store cards on company level. Changing this setting without doing this will lead to failed payments!</comment>
         </field>
+        <field id="enable_installments" translate="label" type="select" sortOrder="219" showInDefault="1"
```
Comment (rikterbeek) [inline etc/adminhtml/system/adyen_pos_cloud.xml]: Do we need this field, can't we just apply instalments if there are configured?

File: view/frontend/web/template/payment/pos-cloud-form.html [hunk 2]
```diff
@@ -50,38 +50,69 @@
...
+
         <div class="actions-toolbar">
             <div class="primary">
                 <button class="action primary checkout"
                         type="submit"
                         data-bind="
                             click: initiate,
                             attr: {title: $t('Place Order')},
-                            enable: getConnectedTerminals().length > 0
+                            enable: isButtonActive()
```
Comment (rikterbeek) [inline view/frontend/web/template/payment/pos-cloud-form.html]: should we not check if there is any connected terminals to select before making the button active ?
What will happen if there are no connected terminals available ?

File: etc/adminhtml/system/adyen_pos_cloud.xml (hunk 1 above)
Comment (cyattilakiss) [reply etc/adminhtml/system/adyen_pos_cloud.xml]: We could but this is a nice feature just to enable and disable them without losing the instalments configuration

File: view/frontend/web/template/payment/pos-cloud-form.html (hunk 2 above)
Comment (cyattilakiss) [reply view/frontend/web/template/payment/pos-cloud-form.html]: indeed, I will put this check in the isButtonActive function

**Pull request merged**